contracts/core/PositionVault.sol

```diff
@@ -662,4 +641,8 @@ contract PositionVault is Constants, ReentrancyGuard, IPositionVault {
         ConfirmInfo memory confirm = confirms[_posId];
         return (position, order, confirm);
     }
+
+    function getVaultUSDBalance() external view override returns (uint256) {
+        return vault.getVaultUSDBalance();
```

Comment:
why do we need this function here, removing it and replace all usage to `vault.getVaultUSDBalance()` seems gas saving